modules/Dockerfile

```diff
@@ -15,33 +16,43 @@ FROM "${DOCKER_HUB_PROXY}python:3.9-slim-bullseye" as python-build
         libpoppler-cpp-dev \
         libfuzzy-dev \
         libssl-dev \
+        ninja-build \
         && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
     
     RUN mkdir /wheels
     WORKDIR /srv
 
-    RUN if [ ! -z ${MODULES_COMMIT} ]; then \
-        git clone https://github.com/MISP/misp-modules.git /srv/misp-modules && cd /srv/misp-modules && git checkout ${MODULES_COMMIT}; \
-        else git clone --branch ${MODULES_TAG} --depth 1 https://github.com/MISP/misp-modules.git /srv/misp-modules; fi
+    RUN <<-EOF
+            if [ ! -z ${MODULES_COMMIT} ]; then
+                git clone https://github.com/MISP/misp-modules.git /srv/misp-modules && cd /srv/misp-modules && git checkout ${MODULES_COMMIT}
+            else
+                git clone --branch ${MODULES_TAG} --depth 1 https://github.com/MISP/misp-modules.git /srv/misp-modules
+            fi
 
-    RUN cd /srv/misp-modules; \
-        echo "pyeti" >> REQUIREMENTS; \
-        echo "greynoise" >> REQUIREMENTS; \
-        echo "git+https://github.com/abenassi/Google-Search-API" >> REQUIREMENTS; \
-        sed -i 's|apiosintds.*|git+https://github.com/davidonzo/apiosintDS\@misp|g' REQUIREMENTS; \
-        sed -i 's/-e //g' REQUIREMENTS; \
-        pip3 wheel -r REQUIREMENTS --no-cache-dir -w /wheels/
+            cd /srv/misp-modules
+            echo "pyeti" >> REQUIREMENTS
+            echo "greynoise" >> REQUIREMENTS
+            echo "git+https://github.com/abenassi/Google-Search-API" >> REQUIREMENTS
```

Comment:
My understanding is that there are no better forks. At some point we should check if that is still true.